content/docs/dvclive/ml-frameworks/huggingface.md

```diff
@@ -1,7 +1,7 @@
-# Hugging Face
+# Hugging Face Transformers
 
 DVCLive allows you to add experiment tracking capabilities to your
```

Comment:
minor: add a reference here to accelerate. Can be a second sentence - "If you are using Accelerate ..."